packages/core/src/components/StepList/StepList.scss

```diff
@@ -0,0 +1,291 @@
+@import "@cmsgov/design-system-support/src/settings/index";
+
+/*
+Step List
+
+@status beta
+
+A step list represents a user's progression through an application or multi-page
+form. It serves as a table of contents and a way to quickly see where they are
+and what they should be working on next.
+
+Markup:
+<div style="max-width: 628px">
+  <ol class="ds-c-step-list ds-u-margin-top--4">
+    <li class="ds-c-step ds-c-step--completed">
+      <h2 class="ds-c-step__header ds-h3 ds-u-display--flex ds-u-justify-content--between">
+        <div class="ds-c-step__title">Choose a tax year</div>
+        <div>
+          <div class="ds-c-step__completed">
+            <span>Completed</span>
+            <a href="#step-1">
+              Edit
+              <span class="ds-u-visibility--screen-reader">
+                "Choose a tax year"
```

Comment:
This sentence feels uneven. I'd like it to read "Edit tax year" for a clear call to action.